oci/private/image.sh.tpl

```diff
@@ -160,7 +158,12 @@ if [ ${#ENV_EXPANSIONS[@]} -ne 0 ]; then
 fi
 
 if [ -n "$OUTPUT" ]; then
-    "${CRANE}" pull "${REF}" "./${OUTPUT}" --format=oci
+    "${CRANE}" pull "${REF}" "./${OUTPUT}" --format=oci --annotate-ref
+    mv "${OUTPUT}/index.json" "${OUTPUT}/temp.json"
+    "${JQ}" --arg ref "${REF}" '.manifests |= map(select(.annotations["org.opencontainers.image.ref.name"] == $ref)) | del(.manifests[0].annotations)' "${OUTPUT}/temp.json" >  "${OUTPUT}/index.json"
```

Comment:
this is needed because zot creates index.json in the storage directory and adds a new manifest entry for each preventing `crane layout gc` from garbage collecting earlier manifests. I'll work to see if i could come up with a clever trick to prevent this. 

FYI crane registry doesn't suffer from this as it never creates the index.json in the storage directory.